Added missing PAYG page to Reference Guide (#1948)
uyuni-project/uyuni-docs: modules/reference/pages/admin/payg.adoc

[[ref-payg]]
= {Payg}

The menu:Admin[{payg}] section allows you to view and manage {payg} instances.

> Every instance shows its description and connection data.

Reviewer: Each instance...

Author: ```suggestion
A description and connection parameters are displayed for each instance
```

> To add new instance, click [button]``Add {payg}``.

Reviewer 2: ```suggestion
To add new instance, click btn:[Add {payg}].
```
I do not know whether your variant would work, too.
